libs/authorizenet/src/reducers/authorize-net/authorize-net.reducer.ts

```diff
@@ -27,6 +31,19 @@ export function daffAuthorizeNetReducer <T extends DaffAuthorizeNetTokenRequest>
 				loading: false,
 				error: action.payload
 			};
+		case DaffAuthorizeNetActionTypes.CheckAcceptJsAction:
+			return {
```

Comment:
This looks like effect behavior that can be exponentially backedoff and retried, not reducer code..

Reply:
sure, I can do that

Reply:
done